chapters/getting-started.Rmd

```diff
@@ -191,13 +191,27 @@ If it is not, you can install it as follows:
 - *Windows*: Follow the [installation instructions][ubc-mds-make-windows] maintained by the
   Master of Data Science program at the University of British Columbia.
 
-> **conda in the shell on Windows**
+> **conda in the Shell on Windows**
 >
 > If you are using Windows and the `conda` command isn't available at the Bash shell,
 > you'll need to open the Anaconda Prompt program (via the Windows start menu)
 > and run the command `conda init bash` (this only needs to be done once).
 > After that, your shell will be configured to use conda going forward.
 
+> **Software Versions**
+> 
+> Throughout the book,
+> we'll be showing you examples of the output you can expect to see.
+> This output is derived from the following software:
+> Git version 2.29.2, Python version 3.7.6,
```

Comment:
There are a number of updates in this PR which suggest Python 3.9 is being used?

Reply:
Oof. Looks like this is an issue with my conda environments using fresh installs of everything, including python. Anything prior to those last chapters is definitely python 3.7; I can enforce the same for my environments in the packaging chapter.